autoPyTorch/api/base_task.py

```diff
@@ -1259,7 +1274,8 @@ def _init_ensemble_builder(
             EnsembleBuilderManager
 
         """
-        assert self._logger is not None, "logger should be initialised to fit ensemble"
+        if self._logger is None:
+            raise ValueError("logger should be initialised to fit ensemble")
```

Comment:
It is not important, but can we go by American English as you seem to often use American spelling.
`initialised` -> `initialized`

Reply:
```suggestion
            raise ValueError("logger should be initialized to fit ensemble")
```